app/admin/applications/new/page.tsx

```diff
@@ -1,31 +1,39 @@
 "use client";
 
 import { useRouter } from "next/navigation";
-import { useCallback } from "react";
+import { useCallback, useState } from "react";
+import { AppTypes } from "@/db/constants";
 
 export default function NewApplicationPage() {
   const router = useRouter();
+  const [selectedAppType, setSelectedAppType] = useState<string>("");
```

Comment:
instead of `string`, create a union of constant string literals called `AppType` so that we could track

Reply:
It should be derived from the existing `AppTypes`